### PR TITLE
add instructions to run Skype with skype-poll-fix.so automatically

### PR DESCRIPTION
**Pull request by gmelikov**
Clean pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,9 @@ This library is made to reduce Skype CPU consumption by increasing poll (on Linu
 How to use
 ----------
 * Compile the source code with ```make```
-* Run Skype with ```LD_PRELOAD=/path/to/skype-poll-fix.so skype``` on Linux or with ```DYLD_INSERT_LIBRARIES=/path/to/skype-poll-fix.dylib skype``` on Mac OS.
+* Run Skype with ```LD_PRELOAD=/path/to/skype-poll-fix.so skype``` on Linux or with ```DYLD_INSERT_LIBRARIES=/path/to/skype-poll-fix.dylib skype``` on Mac OS
```
Comment (ValdikSS) [inline README.md]: Dot is missing.

**Comment by ValdikSS** [comment]
You don't need to close old request and open a new one, just rebase your branch and force-push it and it would automatically apply it to pull request.

File: README.md (hunk 1 above)
Comment (gmelikov) [inline README.md]: Dot was deleted because i added new line.

**Comment by gmelikov** [comment]
Thanks, make it this way next time.

File: README.md (hunk 1 above)
Comment (ValdikSS) [inline README.md]: Why? I think full stop is necessary here.

**Comment by gmelikov** [comment]
Fixed punctuation, i think you was right, thanks!

**Comment by ValdikSS** [comment]
And now squash (rebase) it please.

**Comment by ValdikSS** [comment]
You did something wrong, full stop is gone again.

**Comment by gmelikov** [comment]
Sorry, now it's ready, thanks for help.

**Comment by ValdikSS** [comment]
Thanks a lot!

**Pull request merged**